test/test_1.py

```diff
@@ -5,4 +5,7 @@ def test_demo():
     def add(a, b):
         # This would be imported from your app
         return a + b
-    assert add(1, 1) == 2
\ No newline at end of file
+    assert add(1, 1) == 2
+
+def test_this_will_fail():
+    assert 0 == 1
```

Comment:
this is obviously a mistake,
```suggestion
    assert 1 == 1
```